### DISCUSSION TITLE
LGPD compliance

### DISCUSSION THREAD

**Discussion by obetomuniz** [plausible/analytics#879] [Anything Else]
Are there plans to be officially LGPD compliance?

For reference: https://gdpr.eu/gdpr-vs-lgpd

**Comment by metmarkosaric** [comment]
thanks @obetomuniz! there's no official way to be approved for these things. 

in many ways all the regulations are very similar so we have taken many steps with Plausible to not collect any personal data and to be as privacy friendly as possible. you can read more here https://plausible.io/privacy-focused-web-analytics. and you can read more about our tracking method here https://plausible.io/data-policy

but as usual we have to tell you that we’re not lawyers and we encourage you to discuss specific issues with your lawyer if you have any concerns
